flocker/provision/_aws.py

```diff
@@ -51,6 +53,32 @@
 INSTANCE_TIMEOUT = 300
 
 
+class EliotLogHandler(logging.Handler):
+    # Whitelist ``"msg": "Params:`` field for logging.
+    _to_log = {"Params"}
```

Comment:
@wallrj Here is an example of what I am seeing in, for instance, `setup-cluster.log` on this branch (after `eliot-prettyprint`):
```
c6a52b21-d3dc-4fcd-ba3f-207cf13661a3 -> /3/2
2016-01-29 17:51:21.875634Z
  message_type: 'flocker:provision:aws:boto_logs'
  args: []
  created: 1454089881.875347
  exc_info: None
  exc_text: None
  filename: 'connection.py'
  funcName: '_mexe'
  levelname: 'DEBUG'
  levelno: 10
  lineno: 903
  module: 'connection'
  msecs: 875.0000000000000
  msg: "Params: {'UserData': u'IyEvYmluL3NoCnNlZCAtaSAnL0RlZmF1bHRzICpyZXF1aXJldHR5L2QnIC9ldGMvc3Vkb2Vycwo=', 'Version': '2014-10-01', 'SecurityGroup.1': 'acceptance', 'MinCount': 1, 'BlockDeviceMapping.1.Ebs.DeleteOnTermination': 
'true', 'ImageId': u'ami-d440a6e7', 'BlockDeviceMapping.1.DeviceName': '/dev/sda1', 'KeyName': 'hybrid-master', 'BlockDeviceMapping.1.Ebs.VolumeSize': 8, 'MaxCount': 3, 'Action': 'RunInstances', 'Placement.AvailabilityZone': 'us-west-2a', 'InstanceType': 'm3.large'}"
  name: 'boto'
  pathname: '/home/avg/work/virtualenvs/flocker/lib/python2.7/site-packages/boto-2.38.0-py2.7.egg/boto/connection.py'
  process: 35877
  processName: 'MainProcess'
  relativeCreated: 6032.483816146851
  thread: 34387087360
  threadName: 'MainThread'
```
I find things like `ActionType` and, for instance, `MaxCount` to be pretty useful.